client/src/components/dashboard/UserDashboard.js

```diff
@@ -2,17 +2,14 @@ import React, { Fragment, useEffect } from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import Spinner from "../layout/Spinner";
-import { Link } from "react-router-dom";
-import { getCurrentUserProfile } from "../../actions/profile";
-import { loadUser } from "../../actions/auth";
+import {loadUser} from "../../actions/auth";
 import { Button, Container, Row, Col } from "react-bootstrap";
 
 const UserDashboard = ({
-  getCurrentUserProfile,
-  auth: { user, isAuthenticated, loading },
+  auth: { user, loading },
 }) => {
   useEffect(() => {
-    getCurrentUserProfile();
+      loadUser(localStorage.getItem("token"));
```

Comment:
Please delete the extra empty space for the indention, I suggest to use the IDE's auto Format function to help to format the code so it will align well.